src/main/java/com/digitalasset/testing/ledger/SandboxRunner.java

```diff
@@ -29,22 +27,18 @@
 
   public SandboxRunner(
       String relativeDarPath,
-      Optional<String> testModule,
-      Optional<String> testScenario,
       Integer sandboxPort,
       boolean useWallclockTime,
       Optional<String> ledgerId,
       Optional<LogLevel> logLevel) {
     this.relativeDarPath = relativeDarPath;
-    this.testModule = testModule;
-    this.testScenario = testScenario;
     this.sandboxPort = sandboxPort;
     this.useWallclockTime = useWallclockTime;
     this.ledgerId = ledgerId;
     this.logLevel = logLevel;
   }
 
-  private List<String> commands() {
+  private List<String> getDamlSandboxStarterCommand() {
```

Comment:
What else would be in a "sandbox runner"? Or pick a simpler name like (and use plural):
```suggestion
  private List<String> sandboxStartCommands() {
```

Reply:
strictly speaking, it is a single command with many parameters...

Reply:
as @jberthold-da says